### PR TITLE
NXP-27209: Sonar cleanup

### PR DESCRIPTION
**Pull request by efge**

### CODE REVIEW

**Comment by efge** [comment]
https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-fguillaume/773/

File: nuxeo-services/nuxeo-platform-actions-core/src/main/java/org/nuxeo/ecm/platform/actions/ActionService.java [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
     private static final String LOG_MIN_DURATION_KEY = "nuxeo.actions.debug.log_min_duration_ms";
 
-    private long LOG_MIN_DURATION_NS = -1 * 1000000;
+    private long LOG_MIN_DURATION_NS = -1_000_000;
```
Comment (troger) [inline nuxeo-services/nuxeo-platform-actions-core/src/main/java/org/nuxeo/ecm/platform/actions/ActionService.java]: Why not using `Duration` as well?

[the remark this answers is not in the document]
Comment (efge) [reply nuxeo-services/nuxeo-platform-actions-core/src/main/java/org/nuxeo/ecm/platform/actions/ActionService.java]: Indeed, and I did other cleanups related to that in this class.

Comment (efge) [reply nuxeo-services/nuxeo-platform-actions-core/src/main/java/org/nuxeo/ecm/platform/actions/ActionService.java]: Indeed.

**Pull request merged**